src/lib/pages/nft-collection-details/index.tsx

```diff
@@ -58,33 +58,33 @@ const CollectionDetailsBody = ({
 }: CollectionDetailQueryParams) => {
   const isMobile = useMobile();
   const navigate = useInternalNavigate();
-  const { isFullTier, isSequencerTier } = useTierConfig();
+  const { isFullTier } = useTierConfig();
 
   const { data: collection, isLoading: isCollectionLoading } =
     useCollectionByCollectionAddress(collectionAddress);
 
-  const dataNftsFull = useNfts(collectionAddress, 6, 0, undefined, isFullTier);
-  const dataNftsSequencer = useNftsSequencer(
+  const { data: nfts, isLoading: isNftsLoading } = useNfts(
     collectionAddress,
     6,
     0,
-    undefined,
-    isSequencerTier
+    undefined
   );
-  const { data: nfts, isLoading: isNftsLoading } = isFullTier
-    ? dataNftsFull
-    : dataNftsSequencer;
 
   const { collectionInfos, isLoading: isCollectionInfosLoading } =
     useCollectionInfos(collectionAddress);
 
-  const { data: activitiesCount } = useCollectionActivitiesCount(
+  const { data: activities } = useCollectionActivities(
     collectionAddress,
-    isFullTier
+    1,
+    0,
+    undefined,
+    { enabled: isFullTier }
   );
-  const { data: mutateEventsCount } = useCollectionMutateEventsCount(
+  const { data: mutateEvents } = useCollectionMutateEvents(
     collectionAddress,
-    isFullTier
+    1,
+    0,
+    { enabled: isFullTier }
   );
```

Comment:
For the sake of effeciciently, let's set the limit to 10 since we're using the same query hook in `ActivitiesFull`